.idea/copyright/camunda.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
```

Comment:
This file has a generic name. If it is only related to Optimize, it might make sense to rename it to something optimize related.

Reply:
what actually is this file?

Reply:
The commit that introduced it is called: "intellij java copyright configuration"

Reply:
This file will be deleted with this PR:
https://github.com/camunda/camunda/pull/20592